turbinia/config/turbinia_config_tmpl.py

```diff
@@ -45,8 +46,13 @@
 # different from the OUTPUT_DIR.
 TMP_DIR = '/tmp'
 
+# Default directory to where logs will be stored. Note for a Kubernetes
+# environment, set this path to the shared path configured for Filestore
+# so that logs are can be easily retrieved from one central location.
+LOG_DIR = '/var/tmp'
+
 # File to log debugging output to.
-LOG_FILE = '%s/turbinia.log' % TMP_DIR
+LOG_FILE = '%s/%s.log' % (LOG_DIR, uname().nodename)
```

Comment:
I think this is okay to do here, although I am thinking about if we need to keep the configuration file clean (aka as close to text file as possible)... but that would mean setting it in some __init__ function which could get complicated as this is used by both server, worker, client and importeed by people using the Turbinia libs. Good for now imo.

Reply:
Yeah having it close to text file makes sense but sounds good for now with room for future improvement

Reply:
Yeah, I think it's good for now too, and I share the thought that we probably want to try to keep this clean when possible.  I could imagine that we might want to make this more portable for other clients in the future.